etc/ssdlc_compliance_report.md

```diff
@@ -0,0 +1,40 @@
+# Mongo C++ Driver SSDLC Compliance Report
+
+## Release Creator
+
+- See [C/CXX Release Info](https://docs.google.com/spreadsheets/d/1yHfGmDnbA5-Qt8FX4tKWC5xk9AhzYZx1SKF4AD36ecY/edit?usp=sharing).
+
+## Process Document
+
+- See [go/how-we-develop-software-doc](http://go/how-we-develop-software-doc).
+
+## Tool used to track third party vulnerabilities
+
+- See [Silk](https://us1.app.silk.security/inventory/asset-group/mongodb____DedupedAssetGroup____60640b8853771efe3af5f78ea37af5d1cdd190df).
+- See [C++ Driver - SSDLC Reports](https://drive.google.com/drive/folders/1q9RI55trFzHlh8McALSIAbT6ugyn8zlO) for release-specific reports.
+
+## Third-Party Dependency Information
+
+- See [etc/augmented.sbom.json](https://github.com/mongodb/mongo-cxx-driver/blob/master/etc/augmented.sbom.json) within the release tarball.
+- See [etc/third_party_vulnerabilities.md](https://github.com/mongodb/mongo-cxx-driver/blob/master/etc/third_party_vulnerabilities.md) within the release tarball.
+- See [C++ Driver - SSDLC Reports](https://drive.google.com/drive/folders/1q9RI55trFzHlh8McALSIAbT6ugyn8zlO) for release-specific reports.
+
+## Static Analysis Findings
+
+- See [C++ Driver - SSDLC Reports](https://drive.google.com/drive/folders/1q9RI55trFzHlh8McALSIAbT6ugyn8zlO) for release-specific reports.
+
+## Security Testing Report
+
+- See [Driver Security Testing Summary](https://docs.google.com/document/d/1y2K_RY4GZVXpQvv4JH_35mSzFRTawNJ3mibpvSBU8H0/edit?usp=sharing). Available upon request for approved partners (contact Rishabh Bisht \<rishabh.bisht@mongodb.com\>).
+
+## Security Assessment Report
+
+- Available upon request for approved partners (contact Rishabh Bisht \<rishabh.bisht@mongodb.com\>).
```

Comment:
```suggestion
```

Suggest removing. The "Security Assessment By an External Team" is a non-goal in [Scope: SSDLC Policy Conformance for Client Libs](https://docs.google.com/document/d/1mXxDdY38nGVKWJhIbH83X6mcHsxllAiunH2hg-NdDsY/) so I expect there is no report to share.

Reply:
Added an explicit note stating this report is not applicable to the CXX Driver. LMK if you would prefer the section be removed entirely instead.